client/js/src/client.ts

```diff
@@ -657,7 +657,7 @@ export function api_factory(
 								host,
 								config.path,
 								true
-							)}/queue/join?${params}`
+							)}/queue/join?${params}${url_params ? "&" + url_params : ""}`
```

Comment:
Should we put `url_params` before `params`. Right now, if `url_params` includes e.g. `fn_index`, it'll override the `param` and cause the request to fail

Reply:
done